spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/TaskExecutionManifest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.core;
+
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import org.springframework.cloud.deployer.spi.core.AppDeploymentRequest;
+import org.springframework.core.style.ToStringCreator;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+/**
+ * Description of an execution of a task including resource to be executed and how it was configured via Spring Cloud
+ * Data Flow
+ *
+ * @author Mark Pollack
+ * @author Michael Minella
+ * @since 2.3
+ */
+@Entity
+@Table(name = "TaskExecutionMetadata")
```

Comment:
I think this is a good time to change the name of the table to `TaskExecutionManifest`.

Reply:
Done